disk.go

```diff
@@ -263,7 +264,16 @@ func (d Disk) Details() string {
 	lfmt := "[%2s  %10s %10s %10s %-16s]\n"
 	buf := fmt.Sprintf(lfmt, "#", "Start", "Last", "Size", "Name")
 
-	for _, p := range d.Partitions {
+	pNums := make([]uint, 0, len(d.Partitions))
+	for n := range d.Partitions {
+		pNums = append(pNums, n)
+	}
+
+	sort.Slice(pNums, func(i, j int) bool { return pNums[i] < pNums[j] })
```

Comment:
lgtm, you could save yourself this dance if you used an int with IntSlice: https://golang.org/pkg/sort/#IntSlice

Not sure if you really need uint or not.

Reply:
yeah, but I don't have an int.  maybe I was being pedantic, but a partition number is an unsigned int, because it can't be < 0. (0 <= n <= 127 ... I think 127).

So I think this is about the shortest dance I can do to sort that.